styles/components/_modal.scss

```diff
@@ -39,4 +39,45 @@
   }
   &__button {
   }
+
+  &__win-heading {
+    font-size: 2.5rem;
+  }
+
+  &__win-image {
+    position: relative;
+    width: 10rem;
+    height: 10rem;
+    border: 2px solid #ffca36;
+    margin: 5rem auto auto;
+    border-radius: 50%;
+  }
+
+  &__win-avatar {
+    position: relative;
+    top: 2rem;
+    width: auto;
+    height: 6rem;
+  }
+
+  &__win-medal {
+    position: absolute;
+    right: -2rem;
+    bottom: -2rem;
+    height: 6rem;
+  }
+
+  &__win-text {
```

Comment:
modal__text(--win?), the same basic text styling in all modals in the game

Reply:
done